### PR TITLE
[distributed] Remove unused variable in test_composability/test_pp_composability.py

### PR DESCRIPTION
**Pull request by rec**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #143191

cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o

### CODE REVIEW

File: test/distributed/_composable/test_composability/test_pp_composability.py [hunk 1]
```diff
@@ -537,12 +537,6 @@ def apply_tp(
             else:
                 pipeline_schedule.step()
 
-            # accumulate losses across pipeline microbatches
-            loss = (
```
Comment (albanD) [inline test/distributed/_composable/test_composability/test_pp_composability.py:541]: this one is most likely a bug as we should call a `.backward()` on this one.
cc @kwen2501 ?

Comment (fegin) [reply test/distributed/_composable/test_composability/test_pp_composability.py:541]: cc., @H-Huang

**Review by mori360 — APPROVED** [review]
Thank you for the fix, loss is unused here.
